linalg/operator.hpp

```diff
@@ -261,6 +261,68 @@ class TimeDependentOperator : public Operator
       return const_cast<Operator &>(dynamic_cast<const Operator &>(*this));
    }
 
+   /** Setup the ODE linear system A(y,t) = (I - gamma J) or A = (M - gamma J).
```

Comment:
I'm kinda surprised to see these methods baked into Operator rather than in the ODE classes.  Do these really belong here?

Reply:
This is one of the two approaches I mentioned for attaching an MFEM linear solver to Sundials. This idea here was to mirror the `ImplicitSolve` function used by MFEM's integrators. Sundials integrators need `ImplicitSetup` in addition to `ImplicitSovle` because they will try to reuse the same linear system across Newton iterations and time steps.

Reply:
It does look like the TimeDependentOperator already has the solve concept in it so this is just going along with that.  I hadn't see that recently and it seems backwards to me, but rolling with the way MFEM is handling the integrators it the best choice.

Reply:
@v-dobrev Julian and I are happy with the new way this is implemented here.  Please have a look to make sure you agree with this approach.  If so we'll review this further.